### DISCUSSION TITLE
Override warning for .osxphotos_export.db

### DISCUSSION THREAD

**Discussion by gyansinha** [RhetTbull/osxphotos#1645] [General]
I've setup a cron job for doing the update but have run into the following issue with the automated task:

```shell
Using last opened Photos library: /Users/*/Pictures/Photos Library.photoslibrary
WARNING: found other export database files in this destination directory branch. This likely means you are attempting to export files into a directory that is either the parent or a child directory of a previous export. Proceeding may cause your exported files to be overwritten.
You are exporting to /Volumes/*/Photos, found .osxphotos_export.db files in:
/Volumes/*/.osxphotos_export.db
Do you want to continue? [y/N]:
```

How do I override the manual prompt to proceed with a "y"?

**Comment by RhetTbull** [comment]
--force
Bypass confirmation prompt. Use with caution.

**Reply by gyansinha** [reply]
Thanks - what is the reason for this warning? If I want to have a nightly export from photos, is there anything wrong in having that .osxphotos_export.db file at the root of the export directory?

**Reply by RhetTbull** [reply]
There's nothing inherently wrong with having an `.osxphotos_export.db` file elsewhere in the directory tree but in *most* cases, it likely means you're exporting to a sub-folder of a previous export and for most use cases, this is probably not desired. If not intentional, this could really mess up the previous export. I know because I've unintentionally done it to myself a couple of times. For example, I exported to `/Volumes/photos` then ran another export to `/Volumes/photos/2024` but if that folder already existed (from the previous export) and contained photos, there might be naming conflicts and OSXPhotos would not have read the original `.osxphotos_export.db` to know which photo mapped to which file. Thus I added this feature to prevent myself from shooting myself in the foot. (A couple other users also had similar issues). If your workflow needs to do this then you can just use the `--force` flag. 

OSXPhotos will also complain and warn you if you try to export into the Photos library itself (users have done this...creates a big mess).
